tests/e2e/specs/auth-client-configuration.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+describe( 'Providing client configuration', () => {
+
+	beforeEach( async() => {
+		await activatePlugin( 'google-site-kit' );
+		await visitAdminPage( 'admin.php', 'page=googlesitekit-splash' );
+	} );
+
+	it( 'Should have disabled button on load', async() => {
+
+		await page.waitForSelector( '#wizard-step-one-proceed' );
+
+		const proceedButton = await page.$$eval( '#wizard-step-one-proceed', el => el.map( x => x.hasAttribute( 'disabled' ) ) );
```

Comment:
`page.$$eval` runs `document.querySelectorAll`, whereas `page.$eval` runs `document.querySelector`, which would be more accurate here (since it's a single element with an ID selector). See [Puppeteer API docs](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageevalselector-pagefunction-args).

Then you could simplify the assertion like this:

```js
expect( await page.$eval( '#wizard-step-one-proceed', ( el ) => el.matches( '[disabled]' ) ) ).toBe( false );
```

Reply:
(The same goes for the other assertions in this file as well)

Reply:
Thanks for this, simplified assertion is in place now. I had read the difference, but was overlooking the double $.